### DISCUSSION TITLE
Glued Hotend

### DISCUSSION THREAD

**Discussion by NuclearPhoenixx** [1coderookie/KobraGoNeoInsights#20] [Show and tell]
Maybe something else for the docs, again feel free to use the photos.

Wanted to have a look at the hotend and on that occasion swap out the PTFE tube for the Creality XS bowden tube. Was having slight stringing/oozing problems with the nozzle and rarely there would be a point of slightly burned filament in the print. When I took apart the hotend from the heatsink this is the first thing that greeted me:

![20231114_213739](https://github.com/1coderookie/KobraGoNeoInsights/assets/17517687/48491e88-97ea-41f6-83dc-66a9a51bb1c1)

A lot of gunk just coming out of the heatbreak from the PTFE tube, which I'm pretty sure is glue to presumably keep the tube in place? This can't be good, right? Having glue that melts at like 60°C or something inside a hotend that can get 200°C and above. Well, I heated everything up, took the PTFE tube out of the heatbreak and unscrewed the nozzle. What immediately followed was black gunk, flakes and crap inside pretty much the entire hotend and the nozzle, which stank really bad. I pretty much instantly threw away the nozzle and tried to get as much of the crap out of the hotend.

This is what the PTFE tube looked like after taking out:
![20231114_220815](https://github.com/1coderookie/KobraGoNeoInsights/assets/17517687/3206e5f2-f7c0-4806-8787-c6c52c11be1b)

The tube was about 41.9mm long, so I figure that wasn't too bad. It was also cut perfectly perpendicular, so okay.
![20231114_220252](https://github.com/1coderookie/KobraGoNeoInsights/assets/17517687/cd3d2fe9-799a-42a4-9770-06f05143b6f0)

The only thing I noticed was a pretty big clog inside the tube. I'm pretty sure that happened when I was using it (as a complete newbie I of course printed their stupid owl) and isn't some manufacturing error, altough it looks weird. It's a perfectly circular constriction inside the tube as you can see in the photo. At that point, filament obviously couldn't go through anymore. I guess that's what my first clog looks like. I'm happy to have found this and I'm looking forward to having a lot less problems now.
![20231114_220754](https://github.com/1coderookie/KobraGoNeoInsights/assets/17517687/4050d458-dfaf-40a9-8596-a3e345dff61a)

I obviously threw out the old PTFE tube, cut an equal length (actually ended up almost +1mm) from the Creality XS bowden tube and put it inside the heatbreak. I couldn't easily unscrew the heatbreak probably because of all the crap, so I just used the old length of tube. I hope it's not going to get loose or something. But even if that'll happen, I'm glad to buy a **clean**, brand new hotend some time in the future. The nozzle was equally hard to screw in for the same reason. I had to heat up the entire assembly mutliple times with the heat gun. Otherwise it would have been impossible to do without destroying the thread. Gonna print with the new assembly today, so if I get a lot more gunk out of my prints now or if some of the residual glue starts to clog stuff, I'll report that as well lol

This was the first time opening up my hotend, so it came like that from the factory. Maybe a good PSA for everyone to watch out for that.

**Comment by 1coderookie** [comment]
Thx :)  
I think I mentioned that glue is being used there, but actually not to keep the PTFE tube in place, but for 'securing' the nozzle and the heatbreak in the heater block. At every new stock hotend I had/have, I was able to just pull out the clean tube - there was no additional stuff being used. But: I had hotends where you could see that stuff (thread glue) sticking out of the threaded areas, just like filament would squish out if assembled wrong, but it was actually a bit more clear-ish looking. So I'd assume that's either that sort of glue you see there (which then should've been hard) or it's some gunk that built up from the tube itself or filament. That black stuff was most likely burnt filament which stuck somewhere.  
That being said, a bit of that thread glue they used still sticks in the thread of the block, which can make it difficult to actually screw new parts in there easily. Besides that, at a few stock heater blocks I inspected, the thread somehow didn't go all the way through the block. Instead, it was like at a certain spot the thread was messed up which made it impossible to actually properly install the parts. It didn't seem like that was glue as I tried to clean it the best I could and I also heated it up, it actually looked 'clean' as well. So maybe it still was glue or it was just a bad manufaction quality (which is a known issue with those super cheap aluminum blocks you can get everywhere anyway).  
If you don't want to switch to an all-metal/bi-metal heatbreak, you can keep using the heatbreak tho, just make sure the thread really is clean and free of glue. I'd then suggest to get yourself a plated copper heater block - I can only say good things about them. The ones I got were made good, no problems with the thread, and they also seem to really have a positive effect on the whole temperature behaviour.

**Reply by NuclearPhoenixx** [reply]
Thanks for your opinion and the assessment. If I need a new part for my hotend, I'll just get one of the plated copper v5 blocks and keep using the heatbreak, because I'm mostly printing PLA and now I have the improved tube anyways.

> I think I mentioned that glue is being used there, but actually not to keep the PTFE tube in place, but for 'securing' the nozzle and the heatbreak in the heater block.

Yeah exactly, you mentioned screw lock glue several times, but I didn't expect there to be THAT much glue all over the hotend that it even comes out the top that much, damn...

> So I'd assume that's either that sort of glue you see there (which then should've been hard) or it's some gunk that built up from the tube itself or filament.

Hm, yeah it definitely was sticky, but mostly solid at room temp. I don't think it was from the filament or the tube because then I wouldn't have been able to sort of melt it with my hot air gun. I maybe heated it up to 60° or 70°C and it was almost completely liquid by that point. But maybe you're right and it's some kind of other gunk. That would've been pretty dramatic tho, for it to ooze out that far from the top of the heatbreak. Anyways, whatever it was, it's fixed now :)
